test/test_logit.py

```diff
@@ -0,0 +1,39 @@
+import hypothesis
```

Comment:
remove file

Reply:
Tests need to be implemented for the methods included in logit.py. I just removed the content of the script but keep the file for further reference.